packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -330,6 +330,7 @@ class ApiGateway {
       if (req.query.format === 'sql') {
         await this.sql4sql({
           query: req.query.query,
+          disablePostProcessing: req.query.disable_post_processing === 'true',
```

Comment:
I see that we have `disable_post_processing` in snake-case defined in the docs... But in other places we have CamelCase.... Maybe we can review and align it?

Reply:
+1 for camelCase. 

I assume we need to follow the style we used previously, and it should be camelCase.

cC @igorlukanin